### ISSUE TITLE
Downloading subtitles from the Internet does not work

### ISSUE THREAD

**Issue by kpk-i710** [dsc-uob/subtitle#1]
When I do the load from the line, everything works, it does not download from the Internet at all.

var url = Uri.parse(
    'https://brenopolanski.github.io/html5-video-webvtt-example/MIB2-subtitles-pt-BR.vtt');
var controller = SubtitleController(
  provider: SubtitleProvider.fromNetwork(url),
);
await controller.initial();

I did everything as in the instructions

**Comment by MuhmdHsn313** [comment]
Please try to write your own `NetworkSubtitleProvider` using like this code while I'm fixing the problem!:

```dart
/// ## Network subtitles
/// A class that deals with subtitle files from the **Internet**, you can provide a download
/// link of subtitle in the constructor function for the purpose of completing its processing.
/// You can call it by [NetworkSubtitleProvider].
///
/// ```dart
///
///   // Using  [NetworkSubtitle] class.
///   NetworkSubtitleProvider netSub = new NetworkSubtitleProvider(Uri.parse('<YOUR SUBTITLE PATH URL>'));
///
/// ```
class NetworkSubtitleProvider extends SubtitleProvider {
  /// The url of subtitle file on the internet.
  final Uri url;
  final SubtitleType? type;

  const NetworkSubtitle(
    this.url, {
    this.type,
  });

  // Replace this method with your code to return a `SubtitleObject`.
  @override
  Future<SubtitleObject> getSubtitle() async {
    // Preparing subtitle file data.
    final _repository = SubtitleRepository.inctance;
    final data = await _repository.fetchFromNetwork(url);

    // Find the current format type of subtitle.
    final ext = fileExtension(url.path);
    final type = this.type ?? getSubtitleType(ext);

    return SubtitleObject(data: data, type: type);
  }
}
```

**Comment by MuhmdHsn313** [comment]
Solved in #8
